index.html

```diff
@@ -2,11 +2,11 @@
 <html>
 <head>
   <title>Data-Viz</title>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="lib/bower_components/angular-material/angular-material.css"
   <link rel="stylesheet" type="text/css" href="css/main.css">
 </head>
-<body ng-app="Data-Viz-App">
-  <div ng-view class="container"></div>
+<body ng-app="Data-Viz-App" ng-view layout="column">
 
```

Comment:
Just a reminder: Your ng-view doesn't have to be in your body tag. If you're worried about your side navbar, you might try having it outside the ng-view. Keep in mind that this will mean that the sidenav will be persistent throughout route changes, but my guess is that's what you want anyway. It'd look something like

```html
<body>
  <md-toolbar> 
    ... 
  </md-toolbar>
  <md-sidenav>
    ...
  </md-sidenav>
  <div ng-view></div>
</body>
```